cookiecutter.json

```diff
@@ -20,4 +20,5 @@
     , "add_django_auth_wall": "y"
     , "add_celery": "n"
     , "add_django_cors_headers": "y"
+    , "pre_commit_install": "y"
```

Comment:
Let's have it as `add_pre_commit` similar to other variables.

Reply:
Makes sense!